frontend/packages/knative-plugin/src/components/add/event-sources/NameValueEditorComponent.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { AsyncComponent } from '@console/internal/components/utils/async';
+
+const NameValueEditorComponent = (props) => (
+  <AsyncComponent
```

Comment:
Better to directly use this in the form. Naming it `NameValueEditorComponent` creates the confusion that this component does something different that the one from internal core package. I see that you've created this one because it is being used in two places. I won't mind having to use AsyncComponent twice but if you don't want that then maybe we should rename this to something else like `AsyncNameValueEditor`

Reply:
Makes sense , I'll update it

Reply:
@rohitkrai03 have renamed to  `AsyncNameValueEditor ` and moved to form as well, PTAL.